src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapTheNewGeneration.java

```diff
@@ -0,0 +1,214 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.AutoplayVuap;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VuapAssertions;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VuapVideos;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestAdsVuapTheNewGeneration extends TemplateNoFirstLoad {
+
+  private static final long MAX_AUTOPLAY_MOVIE_DURATION = 40L;
+
+  private static final String FANDOM_ARTICLE_WESTWORLD_LINK = "http://adeng.fandom.wikia.com/articles/whats-coming-westworld-finale";
+
+  private static final String PROJECT43_TNG_ARTICLE_LINK = "http://project43.wikia.com/wiki/DevTemplates/VUAP/TNG";
+
+  private AdsBaseObject openPageWithVideoInLocalStorage(Page page, String player, String video) {
+    final AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.getUrl(urlBuilder.getUrlForWiki("project43"));
+    JavascriptActions runScript = new JavascriptActions(driver);
+    runScript.execute("localStorage.setItem('" + player + video + ");");
+    ads.getUrl(page);
+    return ads;
+  }
+
+  @Test(groups = "AdsVuapDefaultStateAutoplayOasis",
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "adsVuapTngDesctop")
+  public void vuapDefaultStateShouldStartPlayingAdvertisementAutomatically(Page page, String slot, String videoIframeSelector) {
+    openPageWithVideoInLocalStorage(page, VuapVideos.PORVATA_VAST, VuapVideos.VAST_VIDEO);
```

Comment:
I see that you pass those parameters a lot `VuapVideos.PORVATA_VAST, VuapVideos.VAST_VIDEO`. Maybe you could extract method from it or hardcode those two parameters?